examples/core-hello-world/src/main/resources/application.conf

```diff
@@ -1,8 +1,8 @@
 # This is still the default configuration but it's good to see what can be changed
 mesos-client {
 
-  master-url: "127.0.0.1:5050"
-  master-url: ${?MESOS_MASTER_URL}
+  master-url: [ "127.0.0.1:5050" ]
```

Comment:
Let's put a protocol here (and force a protocol). Maybe another PR?

Reply:
Perhaps also we should change the key name from `master-url` to `master-urls` ?

Reply:
That's a bug from my side. Thanks.